docs/website/docs/user-guides/advanced/debugging-with-openshift-toolkit.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Debugging an Application with OpenShift Toolkit
+sidebar_position: 7
+---
+
+Debugging is an unavoidable part of development, and it can prove even more difficult when developing an application that runs remotely.
```

Comment:
I'd mention containers here, as we can also debug traditional apps that run on remote servers..

```suggestion
Debugging is an unavoidable part of development, and it can prove even more difficult when developing an application that runs remotely in containers.
```